specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/analyzeconversations.json

```diff
@@ -0,0 +1,1430 @@
+            }
+          },
+          "default": {
+            "description": "Error response.",
+            "schema": {
+              "$ref": "common.json#/definitions/ErrorResponse"
+            },
+            "headers": {
+              "x-ms-error-code": {
+                "description": "The error code for specific error that occurred.",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Arbitration result": {
+            "$ref": "./examples/conversations/Orchestrator_arbitration.json"
+          },
+          "Conversation project result": {
+            "$ref": "./examples/conversations/Conversation_project.json"
+          },
+          "Orchestrator direct target result": {
+            "$ref": "./examples/conversations/Orchestrator_direct_target.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "AnalyzeConversationTaskKind": {
+      "type": "string",
+      "description": "Enumeration of supported Conversation tasks.",
+      "enum": [
+        "CustomConversation"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTaskResultsKind": {
+      "type": "string",
+      "description": "Enumeration of supported conversational task results",
+      "enum": [
+        "CustomConversationResult"
+      ],
+      "x-ms-enum": {
+        "name": "AnalyzeConversationTaskResultsKind",
+        "modelAsString": true
+      }
+    },
+    "AnalyzeConversationTask": {
+      "description": "The base class of a conversation input task.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskKind"
+        }
+      }
+    },
+    "AnalyzeConversationTaskResult": {
+      "description": "The base class of a conversation task result.",
+      "discriminator": "kind",
+      "required": [
+        "kind"
+      ],
+      "properties": {
+        "kind": {
+          "$ref": "#/definitions/AnalyzeConversationTaskResultsKind"
+        }
+      }
+    },
+    "CustomConversationalTask": {
+      "description": "The input for a custom conversation task.",
+      "allOf": [
+        {
+          "$ref": "#/definitions/AnalyzeConversationTask"
+        }
+      ],
+      "properties": {
+        "analysisInput": {
+          "$ref": "#/definitions/ConversationAnalysisOptions"
+        },
+        "parameters": {
+          "$ref": "#/definitions/CustomConversationTaskParameters"
+        }
+      },
+      "x-ms-discriminator-value": "CustomConversation",
+      "required": [
+        "analysisInput",
+        "parameters"
+      ]
+    },
+    "CustomConversationTaskParameters": {
+      "type": "object",
+      "description": "Input parameters necessary for a Custom Conversation Task.",
```

Comment:
Through descriptions in the swagger, I see a mix of capitalized names and none, like "Item" vs. "item" even for the same phrasing. I'd go through this and make everything that isn't either a product name or class name lowercase. If it's a class name, I'd actually use the class (model) name as-is without spaces. It will make the SDK doc comments much less confusing.

Reply:
Fixed them